src/components/KTabs/KTabs.vue

```diff
@@ -75,6 +75,11 @@ const props = defineProps({
     type: Boolean,
     default: false,
   },
+  anchorTabindex: {
+    type: String,
+    default: '0',
+    validator: (val: string): boolean => val === '0' || val === '-1',
```

Comment:
question: tabindex is always a number (even though it's passed as a string), why is this typed as a string? I think it might be easier to validate the provided value if the prop itself was a number.

e.g. if the user passes `my-wrong-value` as a string, the validator will error; however, you'd still be passing the invalid value directly to the HTML. I think this should be prevented.

Also, I'm not sure the validator makes sense either given the description of the prop. The logic of the validator prevents me from passing a tabindex of any value other than `0` or `-1`

Reply:
Addressed. Added a check to make sure the passed value is either `0` or `-1` and made the type more strict

Reply:
why can the value only be `0` or `-1`? if we have a prop to customize the tabindex, why can't we allow any value?

Reply:
I forgot it can be other values 😅
Fixed now, accepts value `-1 <= x <= 32767` ([mdn](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex))